### PR TITLE
NFIDW NFID Wallet rpc screen subtitle color [sc-14940]

### PR DESCRIPTION
**Pull request by vitaliipustovyi**

### CODE REVIEW

**Review by dmitriiIdentityLabs — APPROVED** [review]

File: packages/identitykit/package.json [hunk 1]
```diff
@@ -58,8 +58,8 @@
   "scripts": {
     "dev": "npm run build-dev --watch",
     "build": "npm run build-prod && npm run build-dev",
-    "build-prod": "rm -rf ./dist && rollup -c rollup.config.prod.js --bundleConfigAsCjs",
```
Comment (olekid) [inline packages/identitykit/package.json]: why?

File: packages/identitykit/package.json [hunk 2]
```diff
@@ -58,8 +58,8 @@
...
-    "build-dev": "rm -rf ./dist && rollup -c rollup.config.dev.js --bundleConfigAsCjs",
```
Comment (olekid) [inline packages/identitykit/package.json]: same

**Review by olekid — CHANGES_REQUESTED** [review]

**Review by olekid — APPROVED** [review]

File: packages/identitykit/package.json (hunk 1 above)
Comment (olekid) [reply packages/identitykit/package.json]: already outdated

File: packages/identitykit/package.json (hunk 2 above)
Comment (olekid) [reply packages/identitykit/package.json]: already outdated

**Pull request merged**